.github/workflows/chaos.yml

```diff
@@ -0,0 +1,59 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: build-branch
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: 0 */12 * * *
+jobs:
+  build:
+    name: compile
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        java: [ 8, 11 ]
+      fail-fast: false
+    steps:
+      - name: Setup link to SSD
```

Comment:
Just curious: are you sure that root is not on SSD. I know that this secondary partition has more space, but didn't know that it has different hardware (SSD vs HDD).

Reply:
This comes from `post-commit.yml`, where I added this name based on my understanding of the [GitHub-hosted  runner specification](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources).  It says "14 GB of SSD disk space", which matches `/mnt`, but not the other mount, which has more total space but less free of it.